source/less/blocks/working-basics.less

```
@@ -35,4 +53,47 @@
     height: 60px;
     background-color: @corporate-color;
   }
+
+  @media @tablet {
+    max-width: 50%;
+    padding-top: 111px;
```

тут можно использовать calc для большей наглядности откуда такие цифры:
```suggestion
    padding-top: calc(80px + 37px); // 80px - новая высота before, 37пк - отступ
```